spec/support/app_fixtures.rb

```diff
@@ -150,11 +150,58 @@ def empty_function; end
         - packs/organisms/app/services/bug_like/fly.rb
     CONTENTS
 
+    write_file('packs/organisms/package.yml', <<~CONTENTS)
```

Comment:
The new implementation requires that packs exist for us to be able to find what pack a file belongs to, so the tests need to be changed accordingly.

I found the tests a bit convoluted coming back to them, so I want to do some work at some point to clean up `UsePackwerk` tests (making each test have a smaller, inline setup).